### PR TITLE
Add CARMA configuration for TRB scenario

### LINKED ISSUE #283: Add configuration for TRB scenario
### Summary

The TRB scenario uses a specific ScenarioRunner scenario and CARMA route configuration. To avoid interfering with users for the XiL configuration, we should add a new configuration directory for the TRB stuff.

Related Jira story: [CDAR-399](https://usdot-carma.atlassian.net/browse/CDAR-399)

### Reasoning for new functionality

_No response_

[CDAR-399]: https://usdot-carma.atlassian.net/browse/CDAR-399?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### PR DESCRIPTION
**Pull request by adamlm**
# PR Details
## Description

This PR adds a new CARMA configuration specific to the TRB scenario. It is mostly the same as the normal xil_cloud configuration but with modified ScenarioRunner and CARMA route settings. We want to avoid interfering with xil_cloud configuration users.

## Related Issue

Closes #283 
Closes [CDAR-399](https://usdot-carma.atlassian.net/browse/CDAR-399)

## Motivation and Context

The TRB scenario has some specific configurations, so we want to keep them separate.

## How Has This Been Tested?

Manually deployed and ran

## Types of changes

- [x] New feature (non-breaking change that adds functionality)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.
[CARMA Contributing Guide](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) 

[CDAR-399]: https://usdot-carma.atlassian.net/browse/CDAR-399?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by paulbourelly999 — APPROVED** [review]

**Review by paulbourelly999 — APPROVED** [review]

File: ail_vru_uc1_scenario/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+      SENSOR_JSON_FILE_PATH: /home/carma-streets/sensor_configurations/sensors.json
+    volumes:
+      - ./sensor_data_sharing_service/manifest.json:/home/carma-streets/sensor_data_sharing_service/manifest.json
+      - ./sensor_data_sharing_service/logs/:/home/carma-streets/sensor_data_sharing_service/logs/
+      - ./MAP/:/home/carma-streets/MAP/
+      - ./sensor_configurations/:/home/carma-streets/sensor_configurations/
+      - /etc/localtime:/etc/localtime:ro
+      - /etc/timezone:/etc/timezone:ro
+  econolite-virtual-controller:
+    image: usdotfhwastol/econolite-virtual-controller:latest
```
Comment (MishkaMN) [inline ail_vru_uc1_scenario/docker-compose.yml:343]: usdotfhwastoldev/econolite-virtual-controller:develop

File: ail_vru_uc1_scenario/cdasim_config/plan_delegator_config/plan_delegator_params.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# Double: If an upcoming lane change will begin in under this time threshold, a turn signal activation command will be published.
+# Units: Seconds
+duration_to_signal_before_lane_change: 2.5
+
+# Int: The maximum duration that Plan Delegator will wait after calling a tactical plugin's trajectory planning service; if trajectory 
+# generation takes longer than this, then trajectory planning will immediately end for the current trajectory planning iteration.
+# NOTE: It is highly desirable to maintain a timeout of 100 ms or less, but trajectory generation success cannot be guaranteed with this duration
+#       for tactical plugins (primarily cooperative_lanechange) in all test scenarios at this time.
+# Units: Milliseconds
+tactical_plugin_service_call_timeout: 350
```
Comment (MishkaMN) [inline ail_vru_uc1_scenario/cdasim_config/plan_delegator_config/plan_delegator_params.yaml:34]: we don't need this file and folder at all anymore

**Review by MishkaMN — CHANGES_REQUESTED** [review]

File: ail_vru_uc1_scenario/docker-compose.yml (hunk 1 above)
Comment (paulbourelly999) [reply ail_vru_uc1_scenario/docker-compose.yml]: https://github.com/usdot-fhwa-stol/carma-config/pull/304

File: ail_vru_uc1_scenario/SubsystemControllerParams.yaml [hunk 3]
```diff
@@ -46,10 +46,6 @@ guidance:
       unmanaged_required_nodes: [''] # TODO add the controller driver once it is integrated with ROS2
       full_subsystem_required: true
       required_plugins:
-        - /guidance/plugins/route_following_plugin
-        - /guidance/plugins/pure_pursuit_wrapper
-        - /guidance/plugins/inlanecruising_plugin
```
Comment (MishkaMN) [inline ail_vru_uc1_scenario/SubsystemControllerParams.yaml]: We still need these 3 plugins. route/ pure/ inlane

**Review by MishkaMN — CHANGES_REQUESTED** [review]

**Review by MishkaMN — APPROVED** [review]

**Review by MishkaMN — DISMISSED** [review]

**Comment by MishkaMN** [comment]
Says `At least 1 approving review is required by reviewers with write access.`

**Review by paulbourelly999 — APPROVED** [review]

**Review by paulbourelly999 — APPROVED** [review]

**Pull request merged**